### PR TITLE
NEWS: add two news items

### PR DESCRIPTION
**Pull request by mattip**
Add the latest 1.19.2 release and "prepare for python 3.9" news items

### CODE REVIEW

**Comment by mattip** [comment]
Rendered https://deploy-preview-358--numpy.netlify.app/news/. The news page does not seem to be linked from the front page. xref gh-322

File: content/en/news.md [hunk 1]
```diff
@@ -3,6 +3,30 @@ title: News
 sidebar: false
 ---
 
+### Python 3.9 is coming, when will NumPy release binary wheels?
+
+_Sept 14, 2020_ -- Python 3.9 will be released in a few weeks. If you are an
+early adopter of Python versions, you may be dissapointed to find that NumPy
+(and other binary packages like SciPy) will not have binary wheels ready on the
+day of the release. It is a major effort to adapt the build infrastructure to a
+new Python versiontypically takes a few weeks for the packages to appear
```
Comment (seberg) [inline content/en/news.md:12]:
```suggestion
new Python version and it typically takes a few weeks for the packages to appear
```

File: content/en/news.md [hunk 2]
```diff
@@ -3,6 +3,30 @@ title: News
...
+  trying to build from source.
+
+
+### Numpy 1.19.2 release
+
+_Sept 10, 2020_ -- [NumPy
+1.19.2](https://numpy.org/devdocs/release/1.19.2-notes.html) is now available.
+This latest release in the 1.19 series fixes several bugs, prepares for the
+[upcoming Cython 3.x
+release](http://docs.cython.org/en/latest/src/changes.html).  and pins
```
Comment (seberg) [inline content/en/news.md:26]:
```suggestion
release](http://docs.cython.org/en/latest/src/changes.html) and pins
```

**Review by seberg — COMMENTED** [review]
Just noticed these typos, thanks!

**Comment by rgommers** [comment]
> use `--only-binary=numpy` or `--only-binary=:all:` to prevent pip from trying to build from source.

This hints at a bug in our PyPI classifiers I think. Not much we can do about it anymore though, because we can't change the classifiers for released versions anymore.

**Review by rgommers — APPROVED** [review]
LGTM, thanks @mattip!

**Comment by rgommers** [comment]
In it goes!

**Pull request merged**